asr-speaker-diarization.ipynb

```diff
@@ -0,0 +1,236 @@
+   "outputs": [],
+   "source": [
+    "# Creating RecognitionConfig\n",
+    "config = riva.client.RecognitionConfig(\n",
+    "  language_code=\"en-US\",\n",
+    "  max_alternatives=1,\n",
+    "  enable_automatic_punctuation=True,\n",
+    "  enable_word_time_offsets=True,\n",
+    ")\n",
+    "\n",
+    "# Use utility function to add SpeakerDiarizationConfig with enable_speaker_diarization=True\n",
+    "# Value of max_speaker_count in SpeakerDiarizationConfig has no effect as of now. It will be honoured in future.\n",
+    "riva.client.asr.add_speaker_diarization_to_config(config, diarization_enable=True)\n",
+    "\n",
+    "# ASR inference call with Recognize\n",
+    "response = riva_asr.offline_recognize(content, config)\n",
+    "print(\"ASR Transcript with Speaker Diarization:\\n\", response)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df24b66d",
+   "metadata": {},
+   "source": [
+    "ASR transcript is split into multiple results based on speech pauses. For each result, every word in the transcript is assigned a speaker tag indicating which speaker spoke that word."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fda5c652",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pretty print transcript with color coded speaker tags. Black color text indicates no speaker tag was assigned.\n",
+    "for result in response.results:\n",
+    "    for word in result.alternatives[0].words:\n",
+    "        color = '\\033['+ str(30 + word.speaker_tag) + 'm'\n",
+    "        print(color, word.word, end=\"\")\n",
+    "      "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb21c265",
+   "metadata": {},
+   "source": [
+    "This completes the tutorial for using Speaker Diarization with Riva ASR."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64c36fe7",
+   "metadata": {},
+   "source": [
+    "## Go deeper into Riva capabilities\n",
+    "\n",
+    "Now that you have a basic introduction to the Riva ASR APIs, you can try:\n",
+    "\n",
+    "### Additional Riva tutorials\n",
+    "\n",
+    "Checkout more Riva ASR (and TTS) tutorials [here](https://github.com/nvidia-riva/tutorials) to understand how to use some of the advanced features of Riva ASR, including customizing ASR for your specific needs.\n",
+    "\n",
+    "\n",
+    "### Sample applications\n",
+    "\n",
+    "Riva comes with various sample applications. They demonstrate how to use the APIs to build applications such as a [chatbot](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/weather.html), a domain specific speech recognition, [keyword (entity) recognition system](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/callcenter.html), or simply how Riva allows scaling out for handling massive amounts of requests at the same time. Refer to ([SpeechSquad)](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/speechsquad.html) for more information.  \n",
```

Comment:
```suggestion
    "Riva comes with various sample applications. They demonstrate how to use the APIs to build applications such as a [chatbot](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/weather.html), a domain specific speech recognition, [keyword (entity) recognition system](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/callcenter.html), or simply how Riva allows scaling out for handling massive amounts of requests at the same time. Refer to [SpeechSquad](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/speechsquad.html) for more information.  \n",
```

Reply:
These links are broken: https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/weather.html and https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/callcenter.html

Reply:
Fixed